doc/src/conf.py

```diff
@@ -102,7 +102,7 @@
 html_static_path = ['static']
 
 def setup(app):
-    app.add_stylesheet('fonts/PragmataPro.css')
+    app.add_stylesheet('http://cs.cmu.edu/~jmsterli/fonts/PragmataPro.css')
```

Comment:
Hmm... why are we using absolute paths? (Or just that you accidentally committed something?)

Reply:
@favonia I'm hosting the fixed webfonts on my AFS rather than on this repo; the reason is that, I realized that to really comply with Fabrizio's license, I need to place the webfonts behind an htaccess barrier.

Reply:
Perhaps we can get something like http://www.cs.cmu.edu/~redprl/.